config.go

```diff
@@ -38,7 +39,9 @@ type Job struct {
 	ClientCertificatePath    string `yaml:"cert"`
 	ClientCertificateKeyPath string `yaml:"key"`
 	ServerCAPath             string `yaml:"ca"`
+	ExcludePattern        	 string `yaml:"excl"`
```

Comment:
Also okay. I was going by the general pattern of using prepositions (`from`, `to`).